### PR TITLE
Creating extension pack

### PR DESCRIPTION
**Pull request by elahehrashedi**
In Progress.

task: https://github.com/microsoft/vscode-cpptools/issues/5977

### CODE REVIEW

File: ExtensionPack/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Change Log
+
+All notable changes to the "cpptoolspack" extension pack will be documented in this file.
+
+## 0.0.1
```
Comment (bobbrow) [inline ExtensionPack/CHANGELOG.md]: We can start with version 1.0

File: ExtensionPack/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# C/C++ Extension Pack for Visual Studio Code Change Log
+
+
+## Version 0.1.0: September 1, 2020
+### List of extensions:
+* C/C++ (Microsoft)
+* C/C++ Themes (Microsoft)
+* Cmake Tools (Microsoft)
```
Comment (sean-mcmanus) [inline ExtensionPack/CHANGELOG.md]: CMake

File: ExtensionPack/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+* Remote Development extension pack (Microsoft)
+* GitHub Pull Requests and Issues (GitHub)
+* Visual Studio Codespaces (Microsoft)
+* LiveShare Extension Pack (Microsoft)
+
```
Comment (sean-mcmanus) [inline ExtensionPack/CHANGELOG.md:16]: Can we add Better C/C++ Syntax? https://github.com/jeff-hykin/cpp-textmate-grammar

File: ExtensionPack/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+## Version 0.1.0: September 1, 2020
+### List of extensions:
+* C/C++ (Microsoft)
+* C/C++ Themes (Microsoft)
+* Cmake Tools (Microsoft)
+* Remote Development extension pack (Microsoft)
+* GitHub Pull Requests and Issues (GitHub)
+* Visual Studio Codespaces (Microsoft)
```
Comment (sean-mcmanus) [inline ExtensionPack/CHANGELOG.md]: Do the VS Codespace and LiveShare extensions work well with our extension? Has anyone tried?

File: ExtensionPack/README.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# C/C++ Extension Pack
+
+This Extension pack includes a set of popular extensions for C++ development in Visual Studio Code. It includes the following set of Visual Studio Code extensions:
+* [C/C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) (Microsoft)
+* [C/C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools-themes) Themes (Microsoft)
+* [Cmake Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools) (Microsoft)
```
Comment (sean-mcmanus) [inline ExtensionPack/README.md]: CMake

File: ExtensionPack/package.json [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    "publisher": "ms-vscode",
+    "icon": "LanguageCCPP_color_128x.png",
+    "readme": "README.md",
+    "author": {
+      "name": "Microsoft Corporation"
+    },
+    "license": "SEE LICENSE IN LICENSE.txt",
+    "version": "0.1.0",
+    "engines": {
+        "vscode": "^1.48.0"
```
Comment (sean-mcmanus) [inline ExtensionPack/package.json:14]: Why is 1.48 required?

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: ExtensionPack/CHANGELOG.md (hunk 4 above)
Comment (elahehrashedi) [reply ExtensionPack/CHANGELOG.md]: @jureid have you tried them together?

File: ExtensionPack/CHANGELOG.md (hunk 3 above)
Comment (elahehrashedi) [reply ExtensionPack/CHANGELOG.md:16]: @jureid

File: ExtensionPack/package.json (hunk 6 above)
Comment (elahehrashedi) [reply ExtensionPack/package.json:14]: I have to investigate that, I don't think 1.48 is required. but I need to see what is the lowest vscode version for all.

File: ExtensionPack/CHANGELOG.md (hunk 4 above)
Comment (jureid) [reply ExtensionPack/CHANGELOG.md]: Yes I have tried both, they do

File: ExtensionPack/package.json (hunk 6 above)
Comment (elahehrashedi) [reply ExtensionPack/package.json:14]: C/C++ "vscode": "^1.44.0"
CMAke "vscode": "^1.30.0"
GitHUb "vscode": "^1.48.0"

I guess we have to go with 1.48.0

File: ExtensionPack/CHANGELOG.md (hunk 2 above)
Comment (jureid) [reply ExtensionPack/CHANGELOG.md]: We can remove "(Microsoft)" from the names

File: ExtensionPack/CHANGELOG.md (hunk 3 above)
Comment (jureid) [reply ExtensionPack/CHANGELOG.md:16]: Let's remove "(Microsoft)" and "(GitHub)" from the names

File: ExtensionPack/CHANGELOG.md (hunk 4 above)
Comment (jureid) [reply ExtensionPack/CHANGELOG.md]: Also remove (Microsoft) and (GitHub) from here
Capitalize "Extension Pack"

File: ExtensionPack/CHANGELOG.md (hunk 3 above)
Comment (jureid) [reply ExtensionPack/CHANGELOG.md:16]: @sean-mcmanus do you use that extension with ours?

File: ExtensionPack/README.md (hunk 5 above)
Comment (jureid) [reply ExtensionPack/README.md]: I think we can remove the second sentence and add a colon after the first.

"This extension pack includes a set of popular extensions for C++ development in Visual Studio Code:"

File: ExtensionPack/CHANGELOG.md (hunk 3 above)
Comment (sean-mcmanus) [reply ExtensionPack/CHANGELOG.md:16]: Better C/C++ Syntax is a more recent version of the lexical syntax highlighter used by VS Code.

File: ExtensionPack/package.json [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+    "displayName": "C/C++ Extension Pack",
+    "description": "Popular extensions for C++ development in Visual Studio Code.",
+    "publisher": "ms-vscode",
+    "icon": "LanguageCCPP_color_128x.png",
+    "readme": "README.md",
+    "author": {
+      "name": "Microsoft Corporation"
+    },
+    "license": "SEE LICENSE IN LICENSE.txt",
+    "version": "0.1.0",
```
Comment (bobbrow) [inline ExtensionPack/package.json]: Should we just make it version 1.0?

File: ExtensionPack/CHANGELOG.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# C/C++ Extension Pack for Visual Studio Code Change Log
+
+
+## Version 0.1.0: September 1, 2020
```
Comment (bobbrow) [inline ExtensionPack/CHANGELOG.md]: version 1.0?  And the date will be September 14.

**Review by bobbrow — CHANGES_REQUESTED** [review]

File: ExtensionPack/README.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
+# C/C++ Extension Pack
+
+This extension pack includes a set of popular extensions for C++ development in Visual Studio Code:
+* [C/C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools)
+* [C/C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools-themes) Themes
```
Comment (bobbrow) [inline ExtensionPack/README.md]: Should "Themes" be in the hyperlink?

**Review by bobbrow — APPROVED** [review]

**Review by jureid — APPROVED** [review]

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**